crates/iota/src/iota_commands.rs

```diff
@@ -217,6 +217,14 @@ pub enum IotaCommand {
         #[clap(long)]
         epoch_duration_ms: Option<u64>,
 
+        /// Make the fullnode dump executed checkpoints as files to this
+        /// directory. This is incompatible with --no-full-node.
+        ///
+        /// If --with-indexer is set, this defaults to a temporary directory.
+        #[cfg(feature = "indexer")]
```

Comment:
They removed the feature upstream?

Reply:
Yes in https://github.com/MystenLabs/sui/pull/19436, but it's related to more changes